platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/BaseBench.java

```diff
@@ -97,7 +96,6 @@ private static void loadConfig() throws IOException {
                 .withConfigDataType(MetricsConfig.class)
                 .withConfigDataType(CryptoConfig.class);
         configuration = configurationBuilder.build();
-        ConfigurationHolder.getInstance().setConfiguration(configuration);
```

Comment:
This statement made sure the newly built configuration was available globally (not just for the benchmark itself but for all used platform classes like `VirtualMap` and `MerkleDb`). How is it achieved now?

Reply:
Configuration is now passed directly to VirtualMap constructor, as well as MerkleDb data source builder.